config/webpack.config.dev.js

```diff
@@ -291,7 +291,10 @@ module.exports = {
       tsconfig: paths.appTsConfig,
       tslint: paths.appTsLint,
     }),
-  ],
+    new webpack.EnvironmentPlugin({
+    FIREBASE_PROJECT: 'rock-paper-scissors-dev',
```

Comment:
I don't see this project in my firebase console. I'm assuming it's created already?

Reply:
Yup